src/components/BooksContainer.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import BooksList from './BooksList';
 import AddBook from './AddBook';
+import styles from './css/BooksContainer.module.scss';
 
 const BooksContainer = () => (
   <div>
     <BooksList />
+    <hr className={styles.separator} />
     <AddBook />
   </div>
 );
```

Comment:
- [ ] Dear @yuvenalmash, I'm impressed with your initial design, But I believe there are possible improvements we can make. as you can see in the Zeplin design the body color is a bit darker and the books/cards are lighter. I see you use the white color for the body and card and apply a border for the cards, this is not bad but to make your design even better I suggest you do differentiate the color between the body background and the background of the cards.

- [ ] Also, consider making the active navigation item a bit bolder and the inactive one a bit lighter. 
### Zeplin Design 
![Screenshot 2023-01-28 174822](https://user-images.githubusercontent.com/69077061/215273221-0826bc50-dcb3-462c-8a8a-de31dd0e47c2.png)

### Your design
![Screenshot 2023-01-28 174900](https://user-images.githubusercontent.com/69077061/215273223-d8d3964a-9041-4170-8915-c933320b1c62.png)